defaults/config.js

```diff
@@ -51,9 +51,18 @@ module.exports = {
 	// Set the default theme.
 	//
 	// @type     string
-	// @default  "themes/example.css"
+	// @default  "example"
 	//
-	theme: "themes/example.css",
+	theme: "example",
+
+	//
+	// A list of plugins installed as global npm modules
```

Comment:
You mentioned `npm link` in the PR comments, do you confirm that having a local theme and `npm link`ing it loads it correctly? It should, but just checking (you'll notice I haven't tested your PR yet...).

What would be really really could would be to have an `lpm` CLI tool to install our packages and themes, and it would simply be a wrapper around `npm` that installs stuff in the `$LOUNGE_HOME/packages` and makes checks around keywords upon installations to make sure you install the correct stuff.
Another benefit of this: in the future, if someone is running multiple instances on their server (for example, one public and one private), we don't want to share packages.

But I'm probably thinking one step too far 😅 Your work is already an amazing step and none of what I said above is made incompatible with your PR. Just something to keep in mind when extending this to generic packages. :)

Reply:
Wow yeah, no, let's ... leave that discussion to a whole other time shall we?